### DISCUSSION TITLE
Welcome to Voice Chat AI Discussions!

### DISCUSSION THREAD

**Discussion by bigsk1** [bigsk1/voice-chat-ai#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kwisss** [comment]
it keeps talking to itself using ollama, xtts and the webui, you have any idea why?

**Reply by bigsk1** [reply]
no idea but maybe microphone not working? If if it detects silance for 3-5 secs then it is the AI's turn to talk. It sends a default "thank you" , if you are running in windows and using xtts, make sure you have Visual C++ installed and all requirements like downloading the checkpoints and putting then in the correct folder structure.
